### ISSUE TITLE
Add alt text to images on the `What's new` page

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5792]
## Area or Component

[Static Site - What's new](https://dev3.simplereport.gov/using-simplereport/whats-new/)

## Quick Summary

[Images do not have alt text](https://dequeuniversity.com/rules/axe/4.7/image-alt?application=AxeChrome)

### How to Replicate

View images on page, alt text doesn't exist

## Accessibility Violation

[Images must have alt text](https://dequeuniversity.com/rules/axe/4.7/image-alt?application=AxeChrome)

## Action Requested

Add Alt text to images.

## Source Code (optional)

[which file or line is the issue attached to?]: #
